plugin/federation/federation.gotpl

```diff
@@ -163,7 +195,8 @@ func (ec *executionContext) __resolve_entities(ctx context.Context, representati
 					return nil
 				{{ end }}
 			{{ end }}
-		{{- end }}
+		{{ end }}
+	{{- end }}
```

Comment:
😅   templates tags are so hard to match up!!!  Did an `{{ end }}`  sneak in or is there a misalignment that makes it harder to spot where the opening tag is?

Reply:
Hopefully, it's a little more aligned and clear now. I think indenting at the case statements might make things clearer? But not doing that yet.